File: Java_Solutions/[Easy]_Challenge_115.java
import java.util.Scanner;

public class easy_challange_115 {
	private static Scanner userInput = new Scanner(System.in);
	
	public static void main(String[]args){
		int userGuess = 0;
		int correctNum = (int) (Math.random() * 1000);
		System.out.println("I'm thinking of a number between 1 and 999..");
		
		while(userGuess != correctNum)
		{
			System.out.println("Guess my number: ");
			System.out.println(correctNum);
			userGuess = userInput.nextInt();
			
			if(userGuess == correctNum){
				System.out.println("You win! You guessed my number!");
			} else if(userGuess < correctNum){
				System.out.println("Your guess was too low.");
			} else if(userGuess > correctNum) {
				System.out.println("Your guess was too high.");
			} else {
				System.out.println("Error.");
			}
		}
	}
}

File: Python_Solutions/[Easy]_Challenge_115.py
import random

def main():
    print("Welcome to guess-that-number game!\n"
          +"If at any point you would like to quit, please type 'exit'.\n\n"
          +"I have thought of a number, 1-100, see if you can guess it!")
    number = random.randrange(1,101)
    guess = 0

    while(guess != number):
        guess = raw_input("Guess a number: ").lower()
        
        if(guess == "exit"):
            exit()

        if(int(guess) == number):
            print("You guessed it! You win!")
            exit()
        elif(int(guess) > number):
            print("Your number is too high!")
        elif(int(guess) < number):
            print("Your number is too low!")
        else:
            print("Error.")        
main()
